pkg/controller/controller_instance_test.go

```diff
@@ -2606,7 +2606,7 @@ func TestPollServiceInstanceFailureProvisioningWithOperation(t *testing.T) {
 		updatedServiceInstance,
 		v1beta1.ServiceInstanceOperationProvision,
 		startingInstanceOrphanMitigationReason,
-		"",
+		errorProvisionCallFailedReason,
```

Comment:
Code LGTM, but I do have one issue.  I find that [line 2582](https://github.com/kubernetes-incubator/service-catalog/pull/2520/files#diff-34dbc217e78a01dffcd4dfcd5d973d3cL2582) is misleading ("Expected polling queue to have a record of test instance as provisioning should have retried") and makes it harder to understand how the system and this test is supposed to behave.   We polled, the poll returned a status 200 and error message indicating the operation failed.  We requeue so **Orphan Mitigation** will happen, not so that we retry the original operation.  If you agree, could you update the text around this?  I'm thinking something like "Expected polling queue to have a record of test instance to process orphan mitigation.")

Same thing at line [358](https://github.com/kubernetes-incubator/service-catalog/pull/2520/files#diff-34dbc217e78a01dffcd4dfcd5d973d3cL358)

Reply:
@jboyd01 Nice catch! Not sure about the line https://github.com/kubernetes-incubator/service-catalog/pull/2520/files#diff-34dbc217e78a01dffcd4dfcd5d973d3cL358 though, do you want me to rename the test case or what?

Reply:
Dohh!!  My cut and paste mistake I think.    Not line 358 in this file, but https://github.com/kubernetes-incubator/service-catalog/pull/2520/files#diff-61aa99b5295af7c58ae76c6191654922R357 (line 358 in kg/controller/controller_instance_ns_test.go)   It's the same thing but in the NS version.  Thanks Nile!

Reply:
Gotcha! Fixed.